samples/bluetooth/hci_spi/README.rst

```diff
@@ -0,0 +1,46 @@
+Bluetooth: HCI SPI
+##################
+
+Overview
+********
+
+Expose Zephyr Bluetooth Controller support over SPI to another device/CPU using
+the Zephyr SPI HCI transport protocol (similar to BlueNRG).
+
+Requirements
+************
+
+* A board with SPI slave, GPIO and BLE support
```

Comment:
... and while I'd love to add even more 96Boards branding into the mainline README for a generic application, I'm not sure that's appropriate for the high-level requirements, especially since the board-level documentation is going to feature prominent references to this app.